experimental/traffic-portal/src/app/shared/file-utils.service.ts

```diff
@@ -153,4 +153,21 @@ export class FileUtilsService {
 		this.window.open(url, "_blank");
 		URL.revokeObjectURL(url);
 	}
+
+	/**
+	 * Exports File
+	 *
+	 * @param json object to be exported
+	 * @param fileName name for the download file
+	 * @param fileExtension extenstion of export file
+	 */
+	public exportFile(json: object, fileName: string, fileExtension: string): void {
```

Comment:
this unnecessary method should be removed, and the `download` method used instead.